src/index.js

```diff
@@ -0,0 +1,161 @@
+// WAVE 2
+
+const state = {
+  temp: 70,
+};
+
+const increaseTemp = document.getElementById('increase-temp');
+const decreaseTemp = document.getElementById('decrease-temp');
+const convertButton = document.getElementById('convert-temp');
+const garden = document.getElementById('garden');
+const currentTemp = document.getElementById('temp');
+
+const updateByTempFahrenheit = (newTemp) => {
+  if (newTemp >= 80) {
+    temp.style.color = '#EA0009';
```

Comment:
It's considered best practices to assign class names instead of directly modifying the style on an element.  That way if you need to change the site theme there's only one place you need to do it (the stylesheet).